### PR TITLE
Separate BeforeInstallPrompt from automatic prompt

### PR DESCRIPTION
**Pull request by mgiuca**
***
<a href="https://pr-preview.s3.amazonaws.com/mgiuca/manifest/pull/692.html" title="Last updated on Jul 9, 2018, 6:59 AM GMT (144c2f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/692/cc53c75...mgiuca:144c2f5.html" title="Last updated on Jul 9, 2018, 6:59 AM GMT (144c2f5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -413,16 +413,58 @@ <h2>
...
+          MAY run the <a>steps to notify that an install prompt is
           available</a>, to give the site the opportunity to prevent the
           default action (which is to install the application). Alternatively,
           the user agent MAY run the <a>steps to notify that an install prompt
           is available</a> at any time, giving the site the opportunity to show
           a <a>site-triggered install prompt</a> without automatically showing
-          the prompt.
+          the prompt. And, the user agent MAY <a>present an install prompt</a>
+          at any time, without giving the site an opportunity to prevent it (in
+          this case, it is RECOMMENDED that the automated prompt be minimally
```
Comment (kenchris) [inline index.html]: be -> is?

Comment (mgiuca) [reply index.html]: Technically "be" is correct. Given it's hard to read this sentence, probably best to rephrase as "the automated prompt SHOULD be minimally..."

But anyway, I'm just throwing this up as a strawman proposal. I don't intend to commit this in its current form. Just collecting feedback on the proposal.

**Comment by aarongustafson** [comment]
I’m good with this change, especially given our move to make BIP optional.

**Comment by mgiuca** [comment]
Whoa, I forgot about this. Shall we move forward on it? It looks like this resolves the BIP-is-optional thing.

I want to review it properly since it's more than a year old and I had previously written "I'm just throwing this up as a strawman proposal. I don't intend to commit this in its current form."

**Pull request closed without merging**